Readme.asciidoc

```diff
@@ -1,68 +1,116 @@
+|Option                    |Is required? |Description                                                                              |Default option
+|NAMESPACE                 |Yes          |Namespace in which this service was created. Used for setting up clustering configuration.|myproject
+|APPLICATION_NAME          |Yes          |Application name                                                                         |caching-service-app
+|IMAGE                     |Yes          |Image that will be used to bootstrap the service                                         |jboss-dataservices/datagrid-online-services
+|NUMBER_OF_INSTANCES       |Yes          |Number of instances                                                                      |1
+|NUMBER_OF_OWNERS          |No           |Number of owners for given piece of data                                                 |2
+|TOTAL_CONTAINER_MEM       |No           |Total container size (both limits and requests)                                          |512
+|APPLICATION_USER          |Yes          |Username used for authentication                                                         |<none>
+|APPLICATION_USER_PASSWORD |No           |Password used for authentication                                                         |<generated>
+|======================
 
-Depending on your OpenShift version and whether the Service Catalog is turned on or off, you need to spin up a new Caching Service instance in your Project.
+=== Creating a client application
 
-Once it's done, just use a Hot Rod client (or REST) to connect to it. Below is an example:
+Once you instantiate one of the services, you might want to connect to it using a Hot Rod (or REST) client.
 
-```
-Configuration cachingServiceClientConfiguration = new ConfigurationBuilder()
-   .addServer()
-      .host("infinispan-app-hotrod") // This service is created by the template
-      .port(11222) // Standard Hot Rod port
-   .build();
+The easiest way to do it is to use https://github.com/jboss-developer/jboss-jdg-quickstarts/tree/jdg-7.2.x/caching-service[JDG Caching Service Quickstart]. The most important thing to remember is that
+ all services use authentication. The same credentials that were used while creating a service need to be used
+ in the client application. Here's an example:
 
-RemoteCacheManager cachingService = new RemoteCacheManager(cachingServiceClientConfiguration);
-RemoteCache<String, String> defaultCache = cachingService.getCache();
 
-defaultCache.put("test", "test"); // Start playing with the Cache
 ```
+Configuration c = new ConfigurationBuilder()
+      .addServer()
+          .host("caching-service-app")
+          .security().authentication().enable()
+            .username("test")
+            .password("test")
+            .realm("ApplicationRealm")
+            .serverName("caching-service")
+            .saslMechanism("DIGEST-MD5")
+            .saslQop(SaslQop.AUTH)
+      .build();
```

Comment:
Maybe we should also provide an example showing how to configure via properties

Reply:
But that would require playing with DeploymentConfig YAML files (since you should put all those environment variables there). I believe we would complicate this guide... is it worth it?

Reply:
Ok, I left it as is... I believe we will soon deprecate the quickstart once boosters are released.